python/oneflow/framework/docstr/repeat.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import oneflow
+from oneflow.framework.docstr.utils import add_docstr
+
+add_docstr(
+    oneflow.repeat,
+    """
+    oneflow.repeat(x, *sizes)
```

Comment:
```suggestion
    oneflow.repeat(input, *sizes)
```

这里是不是应该是 `input`，和这个方法的真正形参名字一致。 如果确认修改，记得这个docstring里的与 `x` 有关的地方，都要改成 `input`

Reply:
已改，下次注意